python/pip_install/extract_wheels/lib/bazel.py

```diff
@@ -82,10 +87,14 @@ def requirement(name):
            name_key = name.replace("-", "_").replace(".", "_").lower()
            return "{repo}//pypi__" + name_key
 
+        all_whl_requirements = [{whl_requirement_labels}]
```

Comment:
Nit: Put this underneath all_requirements so the macros are separated visually from the static definitions.

Reply:
Done in https://github.com/bazelbuild/rules_python/pull/377/commits/b6886f26b7598ebe1e880230f36500b4c0f5a7b4.